applications/crossbar/src/modules/cb_notifications.erl

```diff
@@ -97,6 +99,19 @@ allowed_methods(?SMTP_LOG, _SMTPLogId) ->
 allowed_methods(?CUSTOMER_UPDATE, ?MESSAGE) ->
     [?HTTP_POST].
 
+-spec authorize(cb_context:context()) -> boolean().
+authorize(Context) ->
+    authorize(Context, cb_context:req_nouns(Context), cb_context:account_id(Context)).
+
+-spec authorize(cb_context:context(), req_nouns(), api_binary()) -> boolean().
+authorize(_Context, [{<<"notifications">>, [?NE_BINARY=_Id]}], 'undefined') ->
+    lager:debug("allowing system notifications request"),
+    'true';
+authorize(Context, [{<<"notifications">>, _}, {<<"accounts">>, [AccountId]}], AccountId) ->
+    cb_simple_authz:authorize(Context);
```

Comment:
@icehess I don't think this is required, the simple_authz will preform it independently due to the map (or fold).  Can you test without this?